pkgs/development/libraries/librdmacm/default.nix

```diff
@@ -10,6 +11,22 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ libibverbs ];
 
+  postInstall = ''
+    mkdir -p $out/lib/pkgconfig
+    cat >$out/lib/pkgconfig/rdmacm.pc <<EOF
+    prefix=$out
+    exec_prefix=\''${prefix}
+    libdir=\''${exec_prefix}/lib
+    includedir=\''${prefix}/include
+
+    Name: RDMA library
+    Version: ${version}
+    Description: Library for managing RDMA connections
+    Libs: -L\''${libdir} -lrdmacm
+    Cflags: -I\''${includedir}
+    EOF
```

Comment:
No, i didn't try but you are right it would be better to have this in upstream sources. I will try to submit pull requests to the related projects when I will have time.